### PR TITLE
Various usability and cleanup w.r.t. nodes Enrollment

### PR DESCRIPTION
**Pull request by Geod24**
Part of #1944

### CODE REVIEW

**Review by hewison-chris — APPROVED** [review]
LGTM

File: source/agora/consensus/protocol/Nominator.d [hunk 1]
```diff
@@ -303,7 +303,7 @@ extern(D):
         // check whether the consensus data is valid before nominating it.
         if (auto msg = this.ledger.validateConsensusData(data))
         {
-            log.fatal("tryNominate(): Invalid consensus data: {}. Data: {}",
```
Comment (kchulj) [inline source/agora/consensus/protocol/Nominator.d:306]: I recently found out that there is no such function ^^

**Review by linked0 — APPROVED** [review]
LGTM

**Comment by hewison-chris** [comment]
Can we merge this yet?

**Pull request merged**